balancer-js/src/modules/pools/pool-types/concerns/linear/exit.concern.ts

```diff
@@ -48,4 +80,144 @@ export class LinearPoolExit implements ExitConcern {
     );
     throw new Error('To be implemented');
   };
+
+  buildRecoveryExit = ({
+    exiter,
+    pool,
+    bptIn,
+    slippage,
+  }: Pick<
+    ExitExactBPTInParameters,
+    'exiter' | 'pool' | 'bptIn' | 'slippage'
+  >): ExitExactBPTInAttributes => {
+    this.checkInputsRecoveryExit({
```

Comment:
Feels like this is kind of redundant?

Reply:
I feel like a lot of these internal methods could/should go to a helper file, because they are pretty similar between concerns.
I'd rather do that on a separate PR, since there are other refactors in parallel to this branch and it may cause too many merge conflicts.
For now, I'll keep this method as it's similar to what is already being implemented on other concerns.